src/parent/pom.xml

```diff
@@ -514,6 +514,26 @@
           <generateGitPropertiesFilename>src/main/resources/git.properties</generateGitPropertiesFilename>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.7.5.201505241946</version>
+        <executions>
+          <execution>
+            <id>pre-unit-test</id>
+            <goals>
+              <goal>prepare-agent</goal>
```

Comment:
This goal has to be run before tests in order to collect the coverage metrics. If we remove this execution, users will have to specify `jacoco:prepare-agent` at the right time in the maven command to make sure the metrics are captured.